catalog/opensearch/catalog-opensearch-source/src/main/java/org/codice/ddf/opensearch/source/OpenSearchSource.java

```diff
@@ -123,9 +123,7 @@
 
   protected final EncryptionService encryptionService;
 
-  protected SecureCxfClientFactory<OpenSearch> factory;
-
-  protected boolean isInitialized = false;
+  private SecureCxfClientFactory<OpenSearch> factory;
```

Comment:
✏️ `@Nullable`

Reply:
✏️ Should `factory ` be initialized to `null`?

Reply:
Will @Nullable be meaningful for a private field? 

`factory` is initialized to null automatically

Reply:
I think @nullable is worthwhile because the IDE will warn you to check for null before invoking calling the `SecureCxfClientFactory` methods.

Reply:
Fixed. Thanks for the explanation :)